### DISCUSSION TITLE
Freeze menu and title bar?

### DISCUSSION THREAD

**Discussion by shenwei356** [thegeeklab/hugo-geekdoc#824] [Q&A]
Hi @xoxys , thanks for this greate theme. I really like it. Is it possible to make the menu column fixed/frozen? I mean keeping its position even after users scroll down a long page. I didn't find any configuration.

Best,
Wei

**Comment by xoxys** [comment]
Hi, no that's not possible out of the box, sorry. If you would like to have this behavior, you would have to do it on your own by using a `static/custom.css` file as described at https://geekdocs.de/features/theming/#color-scheme

**Reply by shenwei356** [reply]
Thanks for the fast reply. I've tried and it worked for the left navigation bar.

```
.gdoc-nav nav {
    position: fixed;
}
```
I'm not familiar with CSS, so after several trials, I still failed to get the title bar fixed. Could you please give me a hint?

**Reply by xoxys** [reply]
In general, a fixed title bar is more or less straight forward, see https://www.w3schools.com/howto/howto_css_fixed_menu.asp for an example. However, things starting to get complicated if you want to have a fixed title bar _and_ sidebar.
